medical_pricelist/gql_mutations.py

```diff
@@ -48,12 +48,10 @@ def create_or_update_pricelist(
     if current_name != incoming_name:
         if isinstance(current_pricelist, ServicesPricelist):
             if check_unique_name_services_pricelist(incoming_name):
-                raise ValidationError(
-                    _("mutation.service_name_duplicated"))
+                raise ValidationError(_("mutation.service_name_duplicated"))
```

Comment:
This is not in scope of this ticket, but can you check this validation bug that you have mentioned? Run it with debugger if possible.